README.md

```diff
@@ -86,7 +86,7 @@ class User
 {
     /**
      * @ORM\Column
-     * @Dictionary(name="my_dictionary")
+     * @Dictionary(name="my_dictionary", multiple=false)
```

Comment:
Since value is `multiple=false` by default, we can skip this modification ? 

However the doc could be updated by specifying this parameter ?

Reply:
It's here for documentation purpose :).

Reply:
It could be an other issue : detailed documentation 😉 

IMO it's not intuitive to know why "multiple" is for

Reply:
which means it's not intuitive. But well, yes, an issue may be open to detail each feature of the bundle.